tools/bq-notifier/install.tf

```diff
@@ -0,0 +1,140 @@
+# This code is a prototype and not engineered for production use.
+# Error handling is incomplete or inappropriate for usage beyond
+# a development sample.
+
+variable "topic" {
```

Comment:
Moving the variables declarations to a separate file would make this easier to read